galaxy_importer/ansible_test/runners/openshift_job.py

```diff
@@ -281,7 +281,11 @@ def get_pods(self):
         params = {'labelSelector': f'job-name={self.name}'}
         r = requests.get(
             self.pods_url, headers=self.auth_header, params=params, verify=self.ca_path)
-        return r.json()['items']
+        try:
+            pods = r.json()['items']
+        except KeyError:
```

Comment:
`r.json()` can also raise ValueError if HTTP response body is not a valid JSON.

Reply:
@cutwater updated